meeting.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Austin IWW - Meeting Info</title>
+</head>
+<body>
+    <h1>Meeting Info</h1>
+    <h2>Current Schedule</h2>
+        <p>Online meetings are currently held every other Monday @ 8:00 pm.</p>
+        <p>The next general meeting will be held on 4/12</p>
+        <!-- If desired, we can look into changing this into a calendar widget or set up a script that updates the date periodically -->
```

Comment:
Probably want to make this pull from an authoritative source (google calendar?) so we don't have to update it every week.  I like the calendar widget idea.